### PR TITLE
Small fix to allow satellite scan to proceed by defaulting to empty

### PR DESCRIPTION
**Pull request by kholdaway**

### CODE REVIEW

**Comment by abaiken** [comment]
If you felt like it, you could add a test case.

File: quipucords/fingerprinter/tests_fp_utils.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+#
+
+"""Test the fact engine utils."""
```
Comment (abaiken) [inline quipucords/fingerprinter/tests_fp_utils.py:12]: 😬 This might be a copy/paste

File: quipucords/fingerprinter/tests_fp_utils.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+from django.test import TestCase
+from fingerprinter.utils import product_entitlement_found
+
+
+class EngineTest(TestCase):
```
Comment (abaiken) [inline quipucords/fingerprinter/tests_fp_utils.py:19]: Same comment as above

File: quipucords/fingerprinter/tests_fp_utils.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    """Tests Utils class."""
+
+    def test_process_network_source(self):
```
Comment (abaiken) [inline quipucords/fingerprinter/tests_fp_utils.py]: This name might be a copy paste

File: quipucords/fingerprinter/tests_fp_utils.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        """Test process network source."""
```
Comment (abaiken) [inline quipucords/fingerprinter/tests_fp_utils.py]: This doc string might be a copy paste

File: quipucords/fingerprinter/tests_fp_utils.py (hunk 1 above)
Comment (kholdaway) [reply quipucords/fingerprinter/tests_fp_utils.py]: Fixed.

File: quipucords/fingerprinter/tests_fp_utils.py (hunk 2 above)
Comment (kholdaway) [reply quipucords/fingerprinter/tests_fp_utils.py]: Fixed

File: quipucords/fingerprinter/tests_fp_utils.py (hunk 3 above)
Comment (kholdaway) [reply quipucords/fingerprinter/tests_fp_utils.py]: Fixed

File: quipucords/fingerprinter/tests_fp_utils.py (hunk 4 above)
Comment (kholdaway) [reply quipucords/fingerprinter/tests_fp_utils.py]: Fixed

**Pull request merged**